src/site/content/en/blog/coop-coep/index.md

```diff
@@ -0,0 +1,300 @@
+---
+title: Making your website "cross-origin isolated" using COOP and COEP
+subhead: >
+  Use COOP and COEP to set up a cross-origin isolated environment and enable
+  powerful features like `SharedArrayBuffer`, `performance.measureMemory()` or
+  the JS Self-Profiling API.
+description: >
+  Some web APIs increase the risk of side-channel attacks like Spectre. To
+  mitigate that risk, browsers offer an opt-in-based isolated environment called
+  cross-origin isolated. Use COOP and COEP to set up such an environment and
+  enable powerful features like `SharedArrayBuffer`,
+  `performance.measureMemory()` or the JS Self-Profiling API.
+authors:
+  - agektmr
+date: 2020-04-13
+updated: 2020-04-14
+hero: browsing-context-group.png
```

Comment:
```suggestion
```

This hero image doesn't look good on web.dev/blog (it's cut off):

![image](https://user-images.githubusercontent.com/4713486/79286177-a4897780-7e74-11ea-94c7-608911de3c5a.png)

Or the page itself:

![image](https://user-images.githubusercontent.com/4713486/79286067-4bb9df00-7e74-11ea-9566-1cbe61a3e44b.png)

I'll just remove it for now and you can add it back later. Note that the blog post won't show up on the web.dev homepage until it has a hero image.

Note also that the hero should have an alt tag. Setting the hero image dimensions to `3200px` by `960px` ensures that it looks good. See here for guidance: https://web.dev/handbook/markup-media/#hero-images

It's also possible to specify a [thumbnail](https://web.dev/handbook/markup-media/#thumbnail-images) image for web.dev/blog if you're having trouble making the hero look good across both pages.